battleship-client/src/test/java/com/academy/solid/nie/client/ui/BoardTest.java

```diff
@@ -24,7 +23,7 @@ public void shouldPlaceShip_whenThereAreNoShips() {
         EventHandler<MouseEvent> handler = Mockito.mock(EventHandler.class);
         board.initialize(handler);
 
-        new Cell(point).addShip(ship);
+        new Cell(point).setShip(ship);
```

Comment:
All calls like `Assert.assertTrue()` should be made just `assertTrue` through static imports.